### PR TITLE
37: confirmation modal component

### PR DESCRIPTION
**Pull request by MaxShelestiuk**
Created a simple confirmation modal component, that can be reused where needed.
It uses loading-spinner when submitted.
Also added usage example in main page.

### CODE REVIEW

File: frontend/src/app/shared/components/confirmation-modal/confirmation-modal.component.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  ngOnInit(): void {
+  }
+
+  handleConfirmationModalSubmit() {
+    this.spinnerService.show();
+    setTimeout(() => {
+      this.confirmationModalData.callbackMethod();
+      this.spinnerService.hide();
+    }, 500);
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/confirmation-modal/confirmation-modal.component.ts]: Why did you add timeout here? It's making our app artificially slower

Comment (MaxShelestiuk) [reply frontend/src/app/shared/components/confirmation-modal/confirmation-modal.component.ts]: Timeout was added for Loading-spinner demonstration.
I suggest to remove it after Demo day.

**Review by OlegKarapysh — COMMENTED** [review]
it is frontend task, so add screenshot

**Comment by MaxShelestiuk** [comment]
> 
ScreenShots:
image
After submit - spinner and diabled buttons
image

**Review by Anatolii-Ivanov — DISMISSED** [review]
Ok, just don't forget to remove timeout

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by alinaSielina — APPROVED** [review]
